src/qt/editor/inviwomainwindow.cpp

```diff
@@ -105,9 +105,89 @@
 
 namespace inviwo {
 
+namespace {
+QString addAppendSuffix(const QString& str, bool append) {
+    if (append) {
+        return QString{str}.append(" (Append)");
+    } else {
+        return str;
+    }
+}
+}  // namespace
+
+class MenuKeyboardEventFilter : public QObject {
+public:
+    MenuKeyboardEventFilter(QObject* parent) : QObject(parent) {}
+
+    // This will not trigger on MacOS, so we also trigger update on hovered and aboutToShow.
+    virtual bool eventFilter(QObject*, QEvent* ev) override {
+        if (ev->type() == QEvent::KeyPress) {
```

Comment:
KeyRelease not needed?

Reply:
probably needed, have not tested on windows...